hound_test.rb

```diff
@@ -0,0 +1,6 @@
+puts 'this is a violation'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.